### ISSUE TITLE
gradient clipping value

### ISSUE THREAD

**Issue by chrisiacovella** [choderalab/modelforge#363]
The gradient clipping value for pytorch lightning is hardcoded in; this should be exposed as a parameter that is set in the .toml file.
